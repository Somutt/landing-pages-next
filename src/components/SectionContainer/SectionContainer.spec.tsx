import { screen } from '@testing-library/react';
import { SectionContainer } from '.';

import { renderTheme } from '../../utils/renderTheme';

describe('<SectionContainer />', () => {
  it('should render SectionContainer correctly', () => {
    const { container } = renderTheme(
      <SectionContainer>
        <h1>Teste</h1>
      </SectionContainer>,
    );
    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
