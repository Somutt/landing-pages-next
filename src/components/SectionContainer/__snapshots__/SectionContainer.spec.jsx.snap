// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionContainer /> should render SectionContainer correctly 1`] = `
.c0 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 4.0rem;
  width: 100%;
}

<div
  class="c0"
>
  <h1>
    Teste
  </h1>
</div>
`;
